dev-ml/camlp4/camlp4-4.04_p1.ebuild

```diff
@@ -1,10 +1,10 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI="5"
```

Comment:
I don't understand why you're changing irrelevant things. Most importantly, you're changing somebody else's coding style.

Reply:
done